libraries/wallet/wallet.cpp

```diff
@@ -2039,6 +2040,72 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
             return my->_remote_database_api->get_transaction( id );
         }
 
+        vector<extended_message_object> wallet_api::get_inbox(const inbox_a& param) {
+            FC_ASSERT( !is_locked() );
+            vector<extended_message_object> result;
+            auto remote_result = my->_remote_private_message->get_inbox(param);
+            for( const auto& item : remote_result.inbox ) {
+                result.emplace_back( item );
+                mesage_body tmp = try_decrypt_message( item );
+                result.back().encrypted_message = tmp;
```

Comment:
Here better will be: `result.back().encrypted_message = std::move(tmp);`

Reply:
You right. Done.